src/Sets/HParallelotope.jl

```diff
@@ -298,7 +298,7 @@ The list of constraints of `P`.
 function constraints_list(P::HParallelotope{N}) where {N<:Real}
     D, c = P.directions, P.offset
     n = dim(P)
-    clist = Vector{LinearConstraint{N}}(undef, 2n)
+    clist = Vector{LinearConstraint{N, Vector{N}}}(undef, 2n)
```

Comment:
@mforets: I think you need to use the vector type of `H` here instead.

Reply:
I don't know how to get the vector type of `MN`.

Reply:
Ah, I see the problem. Still, it is not nice that a sparse matrix gets converted to dense vectors. Maybe just do `typeof(D[1, :])`?

Reply:
Yes, i thought about this in the past but discarded the idea. If you also think it's fine, i'll add it.